packages/app/src/cli/services/function/build.ts

```diff
@@ -319,3 +396,30 @@ JavaScript exports with camelCase names are automatically mapped to kebab-case W
 
   return withExport.map((target) => target.export!)
 }
+
+async function installBinary(bin: DownloadableBinary) {
+  const isInstalled = await fs.promises
+    .stat(bin.path)
+    .then(() => true)
+    .catch(() => false)
+  if (isInstalled) {
+    return
+  }
+
+  const url = bin.downloadUrl(process.platform, process.arch)
+  outputDebug(`Downloading ${bin.name} ${bin.version} from ${url} to ${bin.path}`)
+  await fs.promises.mkdir(bin.directory, {recursive: true})
```

Comment:
We don't use `fs` directly, we have a wrapper to make sure it works across all OSes.

For this use our `await mkDir(...)`